dataland-backend/src/main/resources/application.properties

```diff
@@ -12,6 +12,7 @@ dataland-edc-client.baseUrl=http://dataland-edc:9191/api
 spring.security.oauth2.resourceserver.jwt.issuer-uri=https://${PROXY_PRIMARY_URL:local-dev.dataland.com}/keycloak/realms/datalandsecurity
 spring.security.oauth2.resourceserver.jwt.jwk-set-uri=http://keycloak:8080/keycloak/realms/datalandsecurity/protocol/openid-connect/certs
 dataland.authorization.publiclinks=/actuator/health,/actuator/health/ping,/actuator/info,/swagger-ui/**,/v3/api-docs/**,/companies/**,/data/**,/metadata,/metadata/**
+dataland.authorization.internallinks=/internal/nonpersisted,/internal/nonpersisted/**
```

Comment:
why not 
```
dataland.authorization.internallinks=/internal/**
```

Reply:
I think that would also work. Will change it.

Reply:
ok